pkg/manager/member/tikv_member_manager.go

```diff
@@ -40,7 +40,8 @@ import (
 
 const (
 	// tikvClusterCertPath is where the cert for inter-cluster communication stored (if any)
-	tikvClusterCertPath = "/var/lib/tikv-tls"
+	tikvClusterCertPath   = "/var/lib/tikv-tls"
+	tikvStoreLimitPattern = `%s-tikv-\d+\.%s-tikv-peer\.%s\.svc\:\d+`
```

Comment:
it is nearly hardcoded, is there a better way? or can we add a TODO that indicates we need a better way in the future?

Reply:
This method has been used before in this request : https://github.com/pingcap/tidb-operator/pull/1621
But I'm ok to add comment to indicate find a better way in the future.

Reply:
updated.